pkg/cluster/nodes/create.go

```diff
@@ -75,6 +75,11 @@ func CreateControlPlaneNode(name, image, clusterLabel, listenAddress string, por
 		cache.ports = map[int32]int32{kubeadm.APIServerPort: port}
 	})
 
+	// to avoid the DNS crash we need copy host's /etc/resolv.conf to node
+	// ref: https://github.com/kubernetes-sigs/kind/pull/484#issuecomment-489469044
+
+	node.CopyTo("/etc/resolv.conf", "/kind/resolv.conf")
```

Comment:
we should filter the loopback addresses instead of copying it directly, or people that have loopback addresses in their resolv.conf will hit the DNS crass due to the loop.

Reply:
thanks will update

Reply:
from the errors found in the logs, it seems that has to be copied in all nodes, control plane and workers too

> ay 06 11:05:34 kind-worker kubelet[200]: E0506 11:05:34.447977     200 kuberuntime_manager.go:693] createPodSandbox for pod "weave-net-khpnb_kube-system(0ebd9f5c-6fec-11e9-8010-0242ac120002)" failed: open /kind/resolv.conf: no such file or directory

Reply:
Addressed. Thanks. :+1: